portfolio/src/main/webapp/mapScript.js

```diff
@@ -167,9 +167,103 @@ function initMap() {
   map.mapTypes.set('styled_map', styledMapType);
   map.setMapTypeId('styled_map');
 
-  // Set markers on the map.
-  let markerJapanTemple = new google.maps.Marker({position: japanTemple, map: map});
-  let markerNewZealand = new google.maps.Marker({position: newZealandLake, map: map});
-  let markerUganda = new google.maps.Marker({position: ugandaView, map: map});
-  console.log("Initialised the map");
+  // Set markers on the map if marker flag is True.
+  if (markerFlag == true) {
```

Comment:
I believe `bool` variables could be referred in `if` statements without conditions, like:
```javascript
if (markerFlag) {
```